notebooks/tutorials/1_getting_started_lesson2/notebook.ipynb

```diff
@@ -0,0 +1,173 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "2014-01-02 00:00:00+00:00   -0.014137\n",
+       "2014-01-03 00:00:00+00:00   -0.022027\n",
+       "2014-01-06 00:00:00+00:00    0.005376\n",
+       "2014-01-07 00:00:00+00:00   -0.007200\n",
+       "2014-01-08 00:00:00+00:00    0.006406\n",
+       "2014-01-09 00:00:00+00:00   -0.012861\n",
+       "2014-01-10 00:00:00+00:00   -0.006674\n",
+       "2014-01-13 00:00:00+00:00    0.005043\n",
+       "2014-01-14 00:00:00+00:00    0.020123\n",
+       "2014-01-15 00:00:00+00:00    0.020079\n",
+       "Freq: C, Name: Equity(24 [AAPL]), dtype: float64"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Research environment functions\n",
+    "from quantopian.research import returns, symbols\n",
+    "\n",
+    "# Select a time range to inspect\n",
+    "period_start = '2014-01-01'\n",
+    "period_end = '2014-12-31'\n",
+    "\n",
+    "# Query returns data for AAPL\n",
+    "# over the selected time range\n",
+    "aapl_returns = returns(\n",
+    "    assets=symbols('AAPL'),\n",
+    "    start=period_start,\n",
+    "    end=period_end,\n",
+    ")\n",
+    "\n",
+    "# Display first 10 rows\n",
+    "aapl_returns.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Alternative Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition to pricing and volume data, Quantopian integrates a number of alternative datasets that include corporate fundamentals, stock sentiment analysis, and macroeconomic indicators to name a few. You can find the complete list of 50+ datasets on Quantopian's [data page](https://www.quantopian.com/data).  \n",
+    "\n",
+    "Our goal in this tutorial will be to build an algorithm that selects and trades assets based on sentiment data, so let's take a look at PsychSignal's [StockTwits Trader Mood](https://www.quantopian.com/data/psychsignal/stocktwits) dataset. This dataset assigns bull and bear scores to stocks each day based on the aggregate sentiment from messages posted on Stocktwits, a financial communications platform.  \n",
+    "\n",
+    "We can start by inspecting the message volume and sentiment score (bull minus bear) columns from the `stocktwits` dataset. We will query the data using Quantopian's Pipeline API, which is a powerful tool you will use over and over again to access and analyze data in Research. You will learn a lot more about the Pipeline API in the next lesson and a [later tutorial](https://www.quantopian.com/tutorials/pipeline). For now all you need to know is that the following code uses a data pipeline to query `stocktwits` and returns data, and plots the results for AAPL:"
```

Comment:
Okay, I would definitely explain `bull` and `bear` and why we are interested in the difference between them.